src/static/assets/css/markdown.css

```diff
@@ -0,0 +1,695 @@
+@font-face {
```

Comment:
Would it be possible to link it as-is, then override some changes using `!important` or some other technique? Or, is there some clsjs/leinengen -> to -> npm bridge that would allow us to install via dependency management at build time?

If not, please add a NOTICE file according to the Broad's third-party licensing policy here: https://broadinstitute.atlassian.net/wiki/display/DSDE/DSDE+Open+Source+Licensing#DSDEOpenSourceLicensing-Redistributing3rdPartyLibraries